eslint.config.js

```diff
@@ -86,6 +86,12 @@ module.exports = defineConfig([
         ],
         pathGroupsExcludedImportTypes: ['react']
       }],
+      'sort-imports': ['error', {
+        ignoreCase: true,
+        ignoreDeclarationSort: true,
+        ignoreMemberSort: false,
+      }],
+      'no-duplicate-imports': 'error',
```

Comment:
These should be the only manual changes, the rest is generated by running lint autofix `npm run lint:js:fix`